__tests__/service/3.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit, inject } from '@angular/core';
+import { TranslocoService } from '@ngneat/transloco';
+
+@Component({
+  selector: 'inject-test',
+  template: ``
+})
+export class InjectTestClass implements OnInit {
+  transloco = inject(TranslocoService);
```

Comment:
Please add use-cases, the service can be injected into a variable defined at the top and used within the class, it doesn't have to be a property.

Reply:
I'm not getting what you mean here, sorry

Reply:
I mean the following:
```ts
const  transloco = inject(TranslocoService);

@Component({
  selector: 'inject-test',
  template: ``
})
export class InjectTestClass implements OnInit {
```

Reply:
You are correct, but the following is valid:
```ts
function injectService() {
  return inject(TranslocoService);
}

@Component({
  selector: 'inject-test',
  template: ``
})
export class InjectTestClass implements OnInit {
 transloco = injectService();
}
```

The question is how easy it would be to add complete support for the function based injection, maybe we can just start with the most basic functionality that you added and add in the future if needed.